src/main/java/io/xpring/xrpl/javascript/JavaScriptWalletFactory.java

```diff
@@ -38,35 +39,54 @@ public String getDefaultDerivationPath() throws JavaScriptLoaderException {
         return getDefaultDerivationPathFunction.execute().asString();
     }
 
-    public JavaScriptWalletGenerationResult generateRandomWallet() {
+    /**
+     * Generate a random Wallet.
+     *
+     * @param isTest Whether the address is for use on a test network.
+     * @return A {WalletGenerationResult} containing the artifacts of the generation process.
+     * @throws XpringKitException If wallet generation fails.
+     */
+    public JavaScriptWalletGenerationResult generateRandomWallet(boolean isTest) {
```

Comment:
Looks like this wouldn't have a backwards compatible equivalent; not sure if that was intentional or not. These change would break binary compatibility.

Reply:
The `JavascriptWallet` and `JavaScriptWalletFactory` is hidden behind the facade of `Wallet`. The purpose of the `Wallet` class is to act as a public API and a facade. 

We could make this class backwards compatible, but `Wallet` is already backwards compatible and always produces a default value for this argument. 

----

Aside, if you think it's weird that `JavaScriptWallet` and `JavaScriptWallet` factory are hidden behind a `Wallet` class, I actually agree with you. We'll clean this up at some point in the future - it was a dumb design decision I made to get a V0 out the door.

Reply:
Good enough for me 👍